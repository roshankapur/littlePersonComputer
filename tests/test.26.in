1010
2010
3204
4300
0
-99999
5