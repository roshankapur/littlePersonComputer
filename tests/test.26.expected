***Start LPC Execution***
***FATAL ERROR - Attempt to divide by zero***
***lpc execution abnormally terminated***

Registers: 
accumulator +0005
program counter 00002
op-code 00032
operand 00004

      0    1    2    3    4    5    6    7    8    9 
 0 1010 2010 3204 4300 0000 0000 0000 0000 0000 0000 
10 0005 0000 0000 0000 0000 0000 0000 0000 0000 0000 
20 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 
30 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 
40 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 
50 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 
60 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 
70 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 
80 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 
90 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 
***End LPC Execution***
